tslint.mssdl.required.json

```diff
@@ -0,0 +1,24 @@
+{
+  "rulesDirectory": "node_modules/tslint-microsoft-contrib",
+  "rules": {
+    "no-banned-terms": true,
+    "no-delete-expression": true,
+    "no-disable-auto-sanitization": true,
+    "no-document-domain": true,
+    "no-duplicate-parameter-names": true,
+    "no-eval": true,
+    "no-exec-script": true,
+    "no-function-constructor-with-string-args": true,
+    "no-octal-literal": true,
+    // The actual SDL list includes no-reserved-keywords currently but this has been deprecated
```

Comment:
Sometimes the json files error out when they have comments in it
Let's make sure it doesn't happen with this one

Reply:
What do you mean "errors out"? Yes, technically comments in JSON files aren't permitted but TSLint specifically allows that :

From https://palantir.github.io/tslint/usage/configuration/

>  tslint.json configuration files may have JavaScript-style // single-line and /* multi-line */ comments in them (even though this is technically invalid JSON). If this confuses your syntax highlighter, you may want to switch it to JavaScript format.

Reply:
TIL!